das/dasrpc/dasRpcServer.go

```diff
@@ -81,7 +116,25 @@ func (serv *DASRPCServer) Store(ctx context.Context, message hexutil.Bytes, time
 }
 
 func (serv *DASRPCServer) GetByHash(ctx context.Context, certBytes hexutil.Bytes) (hexutil.Bytes, error) {
-	return serv.localDAS.GetByHash(ctx, certBytes)
+	rpcGetByHashRequestGauge.Inc(1)
+	start := time.Now()
+	success := false
+	defer func() {
```

Comment:
Pass success into the func as a variable, otherwise it'll always be false

Reply:
From my testing I can see the success metric getting logged correctly, so I think that's not the case:
```arb_das_rpc_getbyhash_success 183```